packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -6243,7 +6239,7 @@ class IndexedSlot<T> {
 
   /// Information to define where the child occupying this slot fits in its
   /// parent's child list.
-  final T value;
+  final T? value;
```

Comment:
this should be non-nullable and places where we create `IndexedSlot<Element>` should be `IndexedSlot<Element?>`.

Reply:
Done